packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -166,6 +189,53 @@ export function PrescribeWorkflow(props: PrescribeProps) {
     ref?.dispatchEvent(event);
   };
 
+  // let's start screening all of the prescriptions we're drafting
+  const screenDraftedPrescriptions = async () => {
+    // start out by getting the treatment id of the prescription we're drafting now -
+    // we'll want to knwo about it so we cn show an alert underneath it, before it gets
+    // added to the order
+    const inProgressDraftedPrescriptionTreatmentId = props.formStore.treatment?.value?.id;
+
+    // and then get the ones already added to the order (but not persisted)
+    const draftedPrescriptions = [...props.formStore.draftPrescriptions.value];
+
+    // if there is one, add in the prescription being created
+    if (inProgressDraftedPrescriptionTreatmentId) {
+      draftedPrescriptions.push({
+        treatment: { id: inProgressDraftedPrescriptionTreatmentId }
+      });
+    }
+
+    // pluck out all the attributes we won't need so we can make a screening request
+    const sanitizedDraftedPrescriptions = draftedPrescriptions.map(
+      ({
+        id,
+        refillsInput,
+        addToTemplates,
+        templateName,
+        catalogId,
+        treatmentId,
+        __typename,
+        ...draftedPrescription
+      }) => {
+        draftedPrescription.treatment = { id: draftedPrescription.treatment.id };
+
+        return { ...draftedPrescription };
+      }
```

Comment:
You don't need to spread this twice. 
```suggestion
        ...draftedPrescription
      }) => {
        return { ...draftedPrescription, treatment: { id: draftedPrescription.treatment.id } };
      }
```

or 

```suggestion
        ...draftedPrescription
      }) => {
        draftedPrescription.treatment = { id: draftedPrescription.treatment.id };
        return draftedPrescription;
      }
```

[I prefer the former, I dont like to mutate stuff like this, but its fine either way]